lib/modules/datasource/maven/index.spec.ts

```diff
@@ -229,6 +281,27 @@ describe('modules/datasource/maven/index', () => {
     expect(httpMock.getTrace()).toMatchSnapshot();
   });
 
+  it('returns releases from an S3 repository', async () => {
+    mockGenericPackage({
+      base: baseUrlS3,
+      html: null,
+      latest: '2.0.0',
+      jars: { '2.0.0': 200 },
+    });
+
+    const res = await get('org.example:package', baseUrlS3);
+
+    expect(res).toMatchSnapshot();
```

Comment:
please use `toEqual` or `toMatchObject` matcher.

Reply:
This is how all the other tests are set up though?

Reply:
We're trying to reduce our reliance on snapshots, which are often "lost in the noise".

Reply:
And here I thought it was quite a cool little feature! :smile: